### PR TITLE
KIC: Add nerdctl from vendor binary

### PR DESCRIPTION
**Pull request by afbjorklund**
Adds `nerdctl`, for interacting with `containerd` and `buildkitd` (when using "containerd" container runtime)

See #13252

```console
$ minikube start --container-runtime containerd
...
$ minikube ssh -- sudo nerdctl -n k8s.io
nerdctl is a command line interface for containerd

Usage:
  nerdctl [flags]
  nerdctl [command]
...
```

Some +10 MB

### CODE REVIEW

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
```
 > [linux/amd64 stage-2 17/45] RUN export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/' | sed 's/armhf/arm-v7/')  && echo "Installing nerdctl..."     && export NERDCTL_BASE_URL="https://github.com/containerd/nerdctl/releases/download/${NERDCTL_VERSION}"     && curl -sSL --retry 5 --output /tmp/nerdctl.tgz "${NERDCTL_BASE_URL}/nerdctl-${NERDCTL_VERSION:1}-linux-${ARCH}.tar.gz"     && tar -C /usr/local/bin -xzvf /tmp/nerdctl.tgz nerdctl     && rm -rf /tmp/nerdctl.tgz     && chmod 755 /usr/local/bin/nerdctl:
#40 0.318 Installing nerdctl...
#40 0.318 /bin/sh: 1: Bad substitution
```

**Comment by medyagh** [comment]
ok-to-build-image

**Comment by afbjorklund** [comment]
`/bin/sh: 1: Bad substitution`

Was using `bash` commands, need to change to `sed`.

File: deploy/kicbase/Dockerfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ RUN git clone -n https://github.com/Mirantis/cri-dockerd && \
 FROM ubuntu:focal-20210401
 
 ARG BUILDKIT_VERSION="v0.9.3"
+ARG NERDCTL_VERSION="v0.15.0"
```
Comment (AkihiroSuda) [inline deploy/kicbase/Dockerfile:39]: Now we have v0.16.0
https://github.com/containerd/nerdctl/releases/tag/v0.16.0

**Comment by klaases** [comment]
ok-to-build-image

**Comment by afbjorklund** [comment]
ok-to-build-image

**Comment by medyagh** [comment]
ok-to-build-image

**Comment by afbjorklund** [comment]
I will re-open this for the next iteration (1.27), once the other more important features are done.

* https://github.com/kubernetes/minikube/issues/13252

**Pull request closed without merging**